frontend/stories/Footer.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Footer from '../components/footer/index';
+
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Footer',
+  component: Footer,
+  decorators: [withKnobs]
+};
+const data = {
+  links: [
+    { text: 'Github', url: 'github.com' },
+    { text: 'Facebook', url: 'github.com' },
+    { text: 'Twitter', url: 'github.com' },
+    { text: 'infot@m3notorship.com', url: 'github.com' }
+  ],
+  about: {
```

Comment:
what would happen it the about text  length is large number?? the layout will be affected badly , try to put a logic to slice it for a specific text length

Reply:
![footer issue](https://user-images.githubusercontent.com/22384077/88042326-0d574e80-cb4c-11ea-91e5-ad365d144bdb.PNG)

Reply:
i don't see the layout affected. or am i missing something here?

Reply:
the text is too long it should shortened ex.(Lorem ipsum.... Read More)

Reply:
I would disagree. 
the length of the text should be controlled from the backend.